tests/sentry/models/test_groupsubscription.py

```diff
@@ -257,15 +270,13 @@ def test_participating_only(self):
         group = self.create_group(project=project)
         user = self.create_user()
         self.create_member(user=user, organization=org, teams=[team])
-        UserOption.objects.set_value(
-            user=user, key="workflow:notifications", value=UserOptionValue.all_conversations
+        NotificationSetting.objects.update_settings(
+            ExternalProviders.EMAIL,
+            NotificationSettingTypes.WORKFLOW,
+            NotificationSettingOptionValues.ALWAYS,
+            user=user,
         )
 
-        user_option_sequence = itertools.count(300)  # prevent accidental overlap with user id
```

Comment:
we don't need to worry about overlap anymore?

Reply:
I don't think there was ever a reason to expect matching IDs to cause a problem. I removed the line because it added complexity and makes us second guess whether overlapping IDs could be a problem.